v0/destinations/vero/transform.js

```diff
@@ -0,0 +1,142 @@
+const set = require("set-value");
+const get = require("get-value");
+const {
+  defaultRequestConfig,
+  CustomError,
+  constructPayload,
+  getValueFromMessage,
+  isDefinedAndNotNull,
+  isDefinedAndNotNullAndNotEmpty
+} = require("../../util");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+
+// This function handles any tag add or removal requests.
+// Ref - https://developers.getvero.com/?bash#tags
+const tagResponseBuilder = async (message, tags, destination) => {
+  const response = defaultRequestConfig();
+  const category = CONFIG_CATEGORIES.TAGS;
+
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
```

Comment:
Put a check here if the payload is in correct format or not. It maybe possible we are expecting it as an array but customer sends a single value.
For example:
We are expecting the integrations object as below:
```
{
  vero:{
    tags:["ProUser","Follower"]
  }
}
```
But the customer is sending something different:
```
{
  vero:{
    tags:"ProUser"
  }
}
```

Reply:
Just added this check in the recent commit. You can find this check at this line(line 76) https://github.com/rudderlabs/rudder-transformer/blob/d9ad873706807ae1cea293f439a060830ab1bb51/v0/destinations/vero/transform.js#L77